lib/get-cache.js

```diff
@@ -4,10 +4,12 @@ module.exports = getCache
 const LRU = require('lru-cache')
 
 function getCache () {
-  return new LRU({
+  const lru = new LRU({
     // cache max. 15000 tokens, that will use less than 10mb memory
     max: 15000,
     // Cache for 1 minute less than GitHub expiry
     maxAge: 1000 * 60 * 59
   })
+
+  return { get: (key) => lru.get(key), set: (key, token) => lru.set(key, token) }
```

Comment:
I don’t think this change is necessary? If anything, it breaks passing a 3rd argument to cache.set?

Reply:
yeah, I didn't like this either ... wanted to hear your feedback on better path (reading other comments now)